Source/Flow/Private/FlowAsset.cpp

```diff
@@ -71,16 +76,45 @@ void UFlowAsset::PostDuplicate(bool bDuplicateForPIE)
 	}
 }
 
+void UFlowAsset::PostLoad()
+{
+	Super::PostLoad();
+
+	// If we removed or moved a flow node blueprint (and there is no redirector) we might loose the reference to it resulting
```

Comment:
This code fixes-up flow assets that were corrupted by the fixed bug in CallOwnerFunction.